cou/apps/base.py

```diff
@@ -205,9 +207,16 @@ def channel_codename(self) -> OpenStackRelease:
     def current_os_release(self) -> OpenStackRelease:
         """Current OpenStack Release of the application.
 
+        Applications that are colocated in a same machine will upgrade all the packages in the
```

Comment:
Would be possible to do something like this in separate class, because like this we are adding more and more complexity to the base, which can end up to complex <==> easy to break.

Reply:
This also can affect services like `cinder`  so I think it should be in the base

Reply:
I think it would be better in some another class used as mixin or as base for application which require this. However, I'll not block this to be merged if others agree that it's not blocking.

Reply:
I agree it would be an important and desirable change, but not a blocking one. I would propose merging this and refactoring at a later date